sdk-workflows/src/main/java/io/dapr/workflows/client/DaprWorkflowClient.java

```diff
@@ -42,4 +43,30 @@ public DaprWorkflowClient(DurableTaskClient innerClient) throws IllegalArgumentE
   public String scheduleNewWorkflow(String workflowName) {
     return this.innerClient.scheduleNewOrchestrationInstance(workflowName);
   }
+
+  //TODO: Overloaded scheduleNewWorkflow implementations
+
+  public OrchestrationMetadata getWorkflowMetadata(String workflowName) {
+    return this.innerClient.getInstanceMetadata(workflowName, false); //TODO: is false what we want?
+  }
+
+  /**
+   * Terminates the workflow associated with the provided instance id.
+   *
+   * @param workflowName name of workflow to terminate
```

Comment:
Name vs ID has significance here.  In temporal terms, Name references to the workflow definition, ID is a reference to the workflow execution.  For most if not all mgmt API methods we likely want the instance ID.